src/main/webapp/script.js

```diff
@@ -168,6 +170,60 @@ function loadProfile() {
   });
 }
 
+//matches.html functions
+function displayMatches() {
+  let userID = getCurrentUserId();
+  if (userID === null) {
+      return;
+  }
+  fetch('/matches?id=' + userID).then(response => response.json()).then((matches) => {
+    console.log(matches);
+    let matchContainer = document.getElementById('matches-container');
```

Comment:
You'll have to try this to see if I'm right, but I think this can also be `const` because you're not changing the memory address of the variable itself.